sys/include/net/ng_sixlowpan/frag.h

```diff
@@ -42,10 +42,10 @@ extern "C" {
 #define NG_SIXLOWPAN_FRAG_SIZE_MASK     (0x07ff)    /**< mask for datagram size */
 
 /**
- * @brief   General and 1st 6LoWPAN fragmentation header
+ * @brief   1st 6LoWPAN fragmentation header
```

Comment:
Nth fragmentation header extends 1st fragmentation header => implementation assumes first that the header is a (general) fragmentation and then devides them up depending on their dispatch.